### PR TITLE
Exclude external content

### PR DESCRIPTION
**Pull request by alanlanglois**
## Detailed purpose of the PR
The mobile app only support course and chapter, before this PR users could see external content. We added a filter both side (mooc / app) to prevent external content to be loaded in the app. 

### Testing strategy

- [x] I ensured my code is tested: manual emulator android / test-unit. 
- [ ] I took care to mention breaking changes and extra libs if appropriate

### CODE REVIEW

File: src/__fixtures__/cards.ts [hunk 1]
```diff
@@ -212,7 +212,83 @@ export const createChapterCard = ({
...
+    completion,
+    title,
+    isAdaptive = false,
+    isNew = false,
+    isFavorite = false,
+    isDone = false,
+    status,
+    stars = 0,
+    authors = [createCardAuthor({})],
+    accessible = true,
```
Comment (Fausto95) [inline src/__fixtures__/cards.ts:229]:
```suggestion
    accessible = true,
    type = 'scorm'
```

File: src/__fixtures__/cards.ts [hunk 2]
```diff
@@ -212,7 +212,83 @@ export const createChapterCard = ({
...
+      {
+        ref: 'them_EJEGNqKn4',
+        label: 'Flawless French',
+      },
+    ],
+    groupsHidden: 'All courses, Flawless French',
+    course: null,
+    authors,
+    authorsListHidden: 'A good guy',
+    type: 'ext',
```
Comment (Fausto95) [inline src/__fixtures__/cards.ts:270]:
```suggestion
    type,
```

**Review by Fausto95 — APPROVED** [review]
We will need to improve the createExternalContentCard function's return data to match the real one used in the web app.
I'm validating this one so we can fix the bug

**Comment by alanlanglois** [comment]
> We will need to improve the createExternalContentCard function's return data to match the real one used in the web app.
> I'm validating this one so we can fix the bug

yep, I have no idea on how I can find those data. 
Is there specs or doc somewhere?

**Comment by Fausto95** [comment]
Fixed by #602

**Pull request closed without merging**